include/RAJA/policy/loop/launch.hpp

```diff
@@ -41,20 +38,32 @@ struct LaunchExecute<RAJA::expt::null_launch_t> {
 
 
 template <>
-struct LaunchExecute<RAJA::expt::seq_launch_t> {
+struct LaunchExecute<RAJA::seq_launch_t> {
 
   template <typename BODY>
-  static void exec(LaunchContext const &ctx, BODY const &body)
+  static void exec(const size_t shared_mem, LaunchContext const &ctx, BODY const &body)
   {
+    char *kernel_local_mem = new char[shared_mem];
```

Comment:
I assume you want uninitialized memory here? I think malloc is a more direct way of getting that.